lib/src/tray_manager.dart

```diff
@@ -171,6 +171,31 @@ class TrayManager {
       resultData['height'],
     );
   }
+
+  /// Gets the bounds of the screen where this tray icon is visible.
+  ///
+  /// TODO(bselwe): Add support for macOS.
+  ///
+  /// @platforms windows
+  Future<Rect?> getScreenBounds() async {
+    final Map<String, dynamic> arguments = {
+      'devicePixelRatio': window.devicePixelRatio,
```

Comment:
We might need to replace this soon, as the [window singleton has been deprecated](https://docs.flutter.dev/release/breaking-changes/window-singleton).

Reply:
@bselwe Can you change this so we use a parameter instead? That way we can eventually pass something like:
``` dart
View.of(context).devicePixelRatio
```

Reply:
Sure, updating!